examples/scene/ui_scene.rs

```diff
@@ -0,0 +1,67 @@
+#![allow(unused)]
+
+//! This example illustrates constructing ui scenes
+use bevy::{
+    ecs::template::template,
+    prelude::*,
+    scene2::prelude::{Scene, SpawnScene, *},
+};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(world: &mut World) {
+    world.spawn(Camera2d);
+    world.spawn_scene(ui());
+}
+
+fn ui() -> impl Scene {
+    bsn! {
+        Node {
+            width: Val::Percent(100.0),
```

Comment:
It would be nice to write `100pct` or even `100%`.

Reply:
hmm. `100.0` seems standard IMO.

Reply:
Nah, `Pct` is too ambiguous. Not sure the bsn parsing implications, but should be pretty straightforward to parse `Val` from `"100%"` or `"10px"`. 
Otherwise I think just bringing `Val` variants in the prelude would be enough.